src/enclave/enclave_init.c

```diff
@@ -41,24 +41,44 @@ static void __sgxlkl_enclave_copy_app_config(
     sgxlkl_app_config_t* app_config,
     const sgxlkl_config_t* sgxlkl_config)
 {
-    int i = 0, j = 0;
+    int i = 0, envc = 0;
     char** envp = NULL;
+    size_t total_size = 0;
 
     app_config->argc = sgxlkl_config->argc;
-    app_config->argv = oe_malloc((app_config->argc + 1) * sizeof(char*));
 
     for (i = 0; i < app_config->argc; i++)
-        app_config->argv[i] = oe_strdup(sgxlkl_config->argv[i]);
-    app_config->argv[i] = NULL;
+        total_size += strlen(sgxlkl_config->argv[i]) + 1;
```

Comment:
we are removing libc dependencies from src/enclave

Can you change this to oe_strlen ?

Same about apply elsewhere.

Reply:
Sure, no problem. This particular piece of code will likely vanish soon, but other bits (like the new json parser) will have to work on the host and in the enclave, so explicitly calling `oe_*` will be a problem on the host sife. On my branch I added a compatibility header that `#defines` the normal names to the `oe_` names, which allows me to compile with and without that header.

Reply:
(done)

Reply:
sounds good.